vue/sbc-common-components/package.json

```diff
@@ -25,6 +25,7 @@
     "country-list": "^2.2.0",
     "keycloak-js": "^9.0.3",
     "launchdarkly-js-client-sdk": "^2.16.1",
+    "lodash.uniqueid": "^4.0.1",
```

Comment:
This is kind of silly because the entire lodash library is a dependency of other packages we depend on, but I'm being explicit :)

Reply:
he he..i am good with the change..but generally moving away from loadash is advised..  es6 alternatives are there for most of the cases..

Reply:
I agree with you.

In this case there are other alternatives, but this was the easy/quick/clean way :)